src/utils/DepositUtils.ts

```diff
@@ -40,6 +41,9 @@ export async function queryHistoricalDepositForFill(
   fill: Fill | SlowFillRequest,
   cache?: CachingMechanismInterface
 ): Promise<DepositSearchResult> {
+  if (isUnsafeDepositId(fill.depositId)) {
```

Comment:
I think this should probably instead return an instance of `DepositSearchResult`:

```
    return {
      found: false,
      code: InvalidFill.DepositIdOutOfRange,
      reason: `Deposit ID ${depositId} is outside of the sequential deposit ID range [0, 2^32 - 1] .`,
    };
```

Reply:
Good call. Done here: https://github.com/across-protocol/sdk/pull/831/commits/aeb9a95db71a669e9d2fe3e77a17e174979c0c6c